src/convert.js

```diff
@@ -29,6 +29,11 @@ Convert.toString = function convertToString (proto, buf) {
     case 132: // sctp
       return buf2port(buf)
 
+    case 53: // dns
+    case 54: // dns4
+    case 55: // dns6
+      return buf2str(buf)
```

Comment:
/dns, /dns4, and /dns6 have only been reserved for near-future use, and it should be /exp-dns, /exp-dns4, /exp-dns6 for now until we're very confident it's right and have gotten some feedback from others too.